corehq/apps/app_manager/urls.py

```diff
@@ -70,7 +70,7 @@
 urlpatterns = [
     url(r'^$', view_app, name='default_app'),
     url(r'^xform/(?P<form_unique_id>[\w-]+)/$', xform_display, name='xform_display'),
-    url(r'^browse/(?P<app_id>[\w-]+)/modules-(?P<module_id>[\w-]+)/forms-(?P<form_id>[\w-]+)/source/$',
+    url(r'^browse/(?P<app_id>[\w-]+)/(?P<form_unique_id>[\w-]+)/source/$',
```

Comment:
I actually meant that as a general comment for all of these. Endpoints like `edit_form_actions` would give anyone editing an app a weird error when they try to save and lose any case management changes.